### DISCUSSION TITLE
IPHONE BREAKPOINTS

### DISCUSSION THREAD

**Discussion by devjkpjulian** [tailwindlabs/tailwindcss#4718] [Ideas]
It would be easier to have a default utility for break points for apple devices. I don't know if it already exists please let me know if it already exists.

**Comment by Wulfheart** [comment]
This would contradict the way it currently works. One set of breakpoints to rule them all.

**Reply by ianwijma** [reply]
Yeah I fully agree, making styling for per device is ALLOT of work IMO.

**Comment by ianwijma** [comment]
I've had a similar idea where I have added a `xs:` breakpoint (360xp). This allows me to apply different styling for the super small iPhone 5/SE (320xp) and anything bigger is being handled by the `sm:` breakpoint.

Not sure if it is a smart idea to adding my `xs:` breakpoint to the default breakpoints, but you can easily add it to the `tailwind.config.js` by following the [following documentation](https://tailwindcss.com/docs/breakpoints#custom-media-queries)

**Comment by devjkpjulian** [comment]
Well the idea here is that with apple devices such as ipad pro is being read as an Large breakpoint, the orientation of the device for ipad pro is defaulted into portrait view, right now I am using customized breakpoints for other apple devices that needs to adjust accordingly. but wouldn't it be nice to have a default utility like 'pixel-2:hidden' something like that where apple devices mostly have pixel 2 ratio so when the page created will automatically adjust according to responsiveness on apple devices.

**Reply by ianwijma** [reply]
I do see your point. Having a real world sample like `pixel-2:` is nicer to look at than `md:`. But md is easier to ready than pixel-2. Especially future proofness. `md:` would work in 10 years, where `pixel-2:` sounds outdated and require a update to existing code bases to update to `pixel-11` for example.

Like I said before, I personally would recommend just changing the breakpoints using aforementioned [documentation](https://tailwindcss.com/docs/breakpoints#custom-media-queries) to change the queries.
